content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst

```diff
@@ -25,81 +27,75 @@ depends on the stage of the manufacturing process.
+All` button to close it.
 
-.. example::
-   If the manufacturing of a table requires four units of a table leg, and two units of the table
-   leg were scrapped during the manufacturing process, the total quantity of table legs consumed
-   will be six: four units used to manufacture the table plus two units scrapped.
+To select an |MO| that has already been closed, navigate to :menuselection:`Manufacturing -->
+Operations --> Manufacturing Orders`, remove the :guilabel:`To Do` filter from the
+:guilabel:`Search...` bar, and then select the desired |MO|.
 
-Scrap components from tablet view
-=================================
+Once closed, click the :guilabel:`Scrap` button at the top of the |MO| to open the :guilabel:`Scrap`
+pop-up window.
 
-Components can also be scrapped from the manufacturing tablet view. To do so, select the
-:guilabel:`Work Orders` tab on a manufacturing order, then click the :guilabel:`📱 (tablet view)`
-icon for a work order.
+Scrap component from Shop Floor
+-------------------------------
 
-.. image:: scrap_manufacturing/tablet-view-icon.png
-   :align: center
-   :alt: The tablet view icon for a work order.
+To scrap a component from the *Shop Floor* module, begin by navigating to :menuselection:`Shop
+Floor`. Then, either click the :guilabel:`⋮ (three vertical dots)` button on an |MO| card, or select
+a work center from the top navigation, and click the :guilabel:`⋮ (three vertical dots)` button on a
+work order card.
 
-With tablet view open, click the :guilabel:`☰ (menu)` button at the top left of the screen, then
-select the :guilabel:`Scrap` button on the :guilabel:`Menu` pop-up window. The :guilabel:`Scrap`
-pop-up window then appears.
+Either method opens the :guilabel:`What do you want to do?` pop-up window. Click the
+:guilabel:`Scrap` button on the window to open the :guilabel:`Scrap` pop-up window.
 
-.. image:: scrap_manufacturing/tablet-scrap-button.png
-   :align: center
-   :alt: The Scrap button on the Menu pop-up window of the manufacturing tablet view.
+Scrap pop-up window
+===================
 
-Finally, select a component from the :guilabel:`Product` drop-down menu and enter the quantity being
-scrapped in the :guilabel:`Quantity` field. Click :guilabel:`Done` to scrap the component.
+After opening the scrap pop-up window using one of the methods :ref:`detailed above
+<scrap_components/scrap-window>`, select the component or finished product being scrapped, from the
+:guilabel:`Product` drop-down menu. In the :guilabel:`Quantity` field, enter the quantity being
+scrapped.
 
-Scrap finished products
-=======================
+By default, the :guilabel:`Source Location` field is set to the warehouse's pre-production location,
+while the :guilabel:`Scrap Location` field is set to the :guilabel:`Virtual Locations/Scrap`
+location. If either the source or scrap location should be changed, select a different location from
```

Comment:
Can also maybe elaborate here again that the "Scrap Location" field designates where the scrapped products go

Reply:
I feel like that would be a bit redundant.